File: Module-CP/Assignments/assignment14/Pass_by_address(array)/q6.c
#include<stdio.h>
typedef struct Date{
   int d;
   int mm;
   int yyyy;
}Date;
void storeDate(Date*,int);
void displayDate(Date*,int);
int main(){
    Date arr[5];

    storeDate(arr,5);
    displayDate(arr,5);
}
void storeDate(Date*arr,int size){
    printf("Enter Details of Date:");
    for(int i=0;i<size;i++){
        printf("Enter day:");
        scanf("%d",&arr[i].d);
        printf("Enter month:");
        scanf("%d",&arr[i].mm);
        printf("Enter year:");
        scanf("%d",&arr[i].yyyy);
    }
}
void displayDate(Date*arr,int size){
      printf("-----Date Details-----");
        for(int i=0;i<size;i++){
        printf("\nDay = %d",arr[i].d);
        printf("\nMonth = %d",arr[i].mm);
        printf("\nYear = %d",arr[i].yyyy);
    }
}

File: Module-Core Java/Assignments/C to Java/Assignment 14/Main/q6.java
import java.util.*;
class Date{
    int day,mon,yr;
}
public class q6{
    public static void main(String[] args) {
        Date d1 = new Date();
        Date d2 = new Date();

        Scanner sc = new Scanner(System.in);
        
        System.out.println("Day");
        d1.day=sc.nextInt();
        System.out.println("Month");
        d1.mon=sc.nextInt();
        System.out.println("Year");
        d1.yr=sc.nextInt();
        System.out.println("Day");
        d2.day=sc.nextInt();
        System.out.println("Month");
        d2.mon=sc.nextInt();
        System.out.println("Year");
        d2.yr=sc.nextInt();

          System.out.println("Day "+d1.day);
          System.out.println("Month " +d1.mon);
          System.out.println("Year "+d1.yr);
          System.out.println("Day "+d2.day);
          System.out.println("Month " +d2.mon);
          System.out.println("Year "+d2.yr);
    }
}